front-end/src/pages/Login.tsx

```
@@ -1,9 +1,141 @@
+  color: #fff;
+  font-size: 1.1rem;
+  font-family: inherit;
+  transition: border 0.2s, box-shadow 0.2s;
+  outline: none;
+  &:focus {
+    border: 1.5px solid #58A6FF;
+    box-shadow: 0 0 0 2px #58A6FF33;
+  }
+`;
+
+const Error = styled.div`
```

Biome (1.9.4)

[error] Do not shadow the global "Error" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)